src/pages/LandingPage/style.ts

```diff
@@ -0,0 +1,37 @@
+import styled from "styled-components";
+
+export const Page = styled.div`
+    display: flex;
+    padding-top: 20vh;
+`
```

Comment:
adiciona ponto e virgula no final, mó agonia kkkkkkkkkkkkkk
```suggestion
export const Page = styled.div`
    display: flex;
    padding-top: 20vh;
`;
```